app/renderer/renderer.py

```diff
@@ -195,3 +174,52 @@ def _augment_item(self, item, global_errors, global_warnings):
                 global_warnings[item.id] = item_result.warnings
                 item.errors = item_result.get_errors()
                 item.warnings = item_result.get_warnings()
+
+    def _build_piping_context(self):
+        piping_context = {
+            "exercise": self._build_exercse_piping_context(),
+            "responses": self._build_responses_piping_context()
+        }
+
+        return piping_context
+
+    def _build_exercse_piping_context(self):
+        '''
+        Build the exercise data from the survey metadata
+        '''
+        start_date = None
+        end_date = None
+        employment_date = None
+        return_by = None
+
```

Comment:
I think we can remove the try/except from this, it's not needed anymore. 

While your at it might as well remove the use of the getters and just use the property